mu-plugins/blocks/google-map/postcss/style.pcss

```diff
@@ -74,8 +73,47 @@
 		background-image: url(../images/separator-dot.svg);
 		background-repeat: no-repeat;
 	}
+
+	& .wporg-google-map__spinner {
+		--local-size: 16px;
+		--local-position: calc(50% - (var(--local-size) / 2));
+
+		width: var(--local-size);
+		height: var(--local-size);
+		position: absolute;
+		left: var(--local-position);
+		top: var(--local-position);
+		color: var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9));
```

Comment:
Where does `--wp-components-color-accent` come from? Is that reliable?

Reply:
Hmm good spotting, I'll check

Reply:
Turns out no. Set it to blueberry and made it configurable in https://github.com/WordPress/wporg-mu-plugins/pull/538/commits/6f063a3ee3fce3011750f640bfea591f02f12487